### DISCUSSION TITLE
I can only view the sec onion node not my whole network

### DISCUSSION THREAD

**Discussion by kingtay87** [Security-Onion-Solutions/securityonion#6362] [General] [answered]
when sec onion is configured and running it will not detect my whole network and i does only one node and that is itself nothing more i tried dmz port forwarding the switch management i have a xb7 xfinity router any ideas are appreciated maybe so-import-pcap?

**Comment by tomjones1977** [comment, marked as answer]
I am assuming you are talking about network traffic with this problem - what setup do you have to send traffic to your SO server?  Do you have a SPAN port enabled on your switch, a network tap?  What is physically connected to your sniffing interface?

**Reply by kingtay87** [reply]
![IMG_20211128_133721.jpg](https://user-images.githubusercontent.com/95152915/143782438-dadc0886-1c09-4ea5-9b0c-2a24c8f59b9e.jpg)![IMG_20211128_133644.jpg](https://user-images.githubusercontent.com/95152915/143782444-50f95b00-b026-4ee1-9314-2c8aba96747a.jpg) it will not monitor all traffic on the network only the sec onion machine I've tried everything and need help

**Reply by tomjones1977** [reply]
The Netgear FS105 is an unmanaged switch, so you won't be able to configure a SPAN/mirror port on there.  That means that it will only be broadcast traffic that gets to your sniffing interface. 

If you want to monitor traffic going between devices on your network you will need either a network tap (expensive) or a managed switch.  For example, the GS108Ev3 has a port mirroring capability and is fairly cheap. 

Without something to get a copy of the packets to your sniffing interface you will be very limited in what you can monitor.

**Reply by kingtay87** [reply]
What about both connections cat5e hooked directly up to my xb7 router

**Reply by kingtay87** [reply]
![IMG_20211128_150448.jpg](https://user-images.githubusercontent.com/95152915/143784083-7dd61a40-8921-4c53-8f3d-cfe71ca12025.jpg)
What about this?????

**Reply by tomjones1977** [reply]
> What about both connections cat5e hooked directly up to my xb7 router

Based on a quick google search it doesn't look like the xb7 has port mirroring either.  

The GS605 in the picture is an unmanaged switch too, so no port mirroring.

The problem you have is that you are running a normal switched network.  Therefore there's no reason for the router or the switches to send packets destined for your LAN clients to the SO sniffing interface.  What you need is a device that will make a copy of the traffic that it sees and send it to security onion.  Without this you can still do things like log forwarding to the management interface, but you will not be able to monitor network traffic properly.
